pkg/apis/servicerevision/extension.go

```diff
@@ -163,7 +163,15 @@ func manageResources(
 	for _, c := range recordRess {
 		k := dao.ServiceResourceGetUniqueKey(c)
 		if observedRessIndex[k] != nil {
+			c.Edges.Instances = observedRessIndex[k].Edges.Instances
+
+			err = dao.ServiceResourceInstancesEdgeSave(ctx, modelClient, c)
```

Comment:
Why is this operation not in the transaction?

Reply:
Updated